trs_filer/ga4gh/trs/endpoints/registerObjects.py

```diff
@@ -0,0 +1,73 @@
+"""Controller for adding new objects."""
+
+from random import choice
+from typing import Dict
+from trs_filer.app import logger
+from flask import (current_app, request)
+from pymongo.errors import DuplicateKeyError
+
+
+class CreateToolPostObject:
+    def __init__(self, request: request):
+        self.db_collection = (
+            current_app.config['FOCA'].db.dbs['trsStore'].
```

Comment:
It's really a minor point, but I think adding the line break _before_ the dot is much cleaner, as it clearly indicates that we are calling a method in that second line (and not an independent call).

So:

```py
(
    my_long_line_object
    .method()
)
```

rather than

```py
(
    my_long_line_object.
    method()  # looks like a function call at first glance
)
```